File: SEBASTIAN_CHAN_00550218/C#/Animales/Mamifero.cs
using System;
public class Mamifero
{
    private string nombre;
    protected double peso;
    
    public Mamifero(string nombre, double peso)
    {
        this.nombre = nombre;
        this.peso = peso;
    }
    
    public void SetNombre(string nombre)
    {
        this.nombre = nombre;
    }
    
    public string GetNombre()
    {
        return this.nombre;
    }
    public void SetPeso(double peso){
        this.peso=peso;
    }
    public double GetPeso(){
        return this.peso;
    }
}

File: SEBASTIAN_CHAN_00550218/C++/Mamifero.cpp
#include "Mamifero.hpp"
#include <string>

using namespace std;

// Constructor
Mamifero::Mamifero(string nombre, double peso) : nombre(nombre), peso(peso) {}

// Métodos
void Mamifero::setNombre(string nombre) {
    this->nombre = nombre;
}

string Mamifero::getNombre() {
    return nombre;
}

// ✅ Implementación del método getPeso()
void Mamifero::setPeso(double peso){
    this->peso=peso;
}
double Mamifero::getPeso() {
    return peso;
}